components/fieldgroup/stories/template.js

```diff
@@ -30,29 +35,31 @@ export const Template = ({
 	>
 		${FieldLabel({
 			...globals,
+			isRequired,
 			size: "m",
-			label: "Field Group Label",
+			label: fieldLabel,
 			alignment: labelPosition === "side" ? "right" : "top",
 		})}
-
 		<div class="${rootClass}InputLayout">
-			${repeat(
-				items,
-				(item) => item.id,
-				(item) => {
-					return Radio({
+			${inputType === "radio" ? items.map((item) =>
```

Comment:
Checkbox uses `@change`, which doesn't play nicely with Lit's `repeat()` directive, so this was changed to `.map()`.